feature/space-entrepreneurship-funding-sources.md

```diff
@@ -5,11 +5,10 @@ menu: true
 order: 4
 ---
 
-We were looking for a directory of funding resources for NewSpace startups and couldn't find one. So we decided to start our own.
+*We were looking for a directory of funding resources for NewSpace startups and couldn't find one. So we decided to start our own.*
```

Comment:
The title is Space Entrepreneurship funding, but here we say `NewSpace startups` which I'd consider a subclass of the former. Should we change one or the other so they match, or something?